src/PKSim.Core/Repositories/IDiseaseStateRepository.cs

```diff
@@ -8,5 +8,7 @@ namespace PKSim.Core.Repositories
    public interface IDiseaseStateRepository : IStartableRepository<DiseaseState>
    {
       IReadOnlyList<DiseaseState> AllFor(SpeciesPopulation population);
+
+      DiseaseState HealthyState { get; }
```

Comment:
Health**y**State or HealthState?

Reply:
This is the ONE instance of HealthyState. Correct here with *Healthy*